models/User.js

```diff
@@ -1,13 +1,14 @@
-"use strict";
-
-var mongoose = require('mongoose');
-var mongodb = require('common/makeCredential')('mongodb');
-var db = mongoose.createConnection(mongodb);
+import mongoose from 'mongoose';
+import mockgoose from 'mockgoose';
+import Promise from 'bluebird';
+import bcrypt from 'bcryptjs';
+import { SALT_WORK_FACTOR } from 'common/constant';
+if (process.env.NODE_ENV === 'test') {
+  mockgoose(mongoose);
+}
 
-var Q = require('q');
-var bcrypt = require('bcryptjs');
-var SALT_WORK_FACTOR = require('common/constant').SALT_WORK_FACTOR;
-var Utils = require('common/Utils');
+const mongodbCredential = process.env.KINDLIZED_MONGODB;
+const db = mongoose.createConnection(mongodbCredential);
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).